### DISCUSSION TITLE
LOG the viewer count at a set period of time.

### DISCUSSION THREAD

**Discussion by josencarnacao** [Chatterino/chatterino2#5591] [Ideas] [answered]
Hi,
Amazing software and love it for a long time.
May there be the LOGging of how many Viewers Count at a given period of time.
Maybe, set by the user.

Cheers from Portugal :)

**Comment by jupjohn** [comment, marked as answer]
This feels out-of-scope of what chatterino aims to offer: it's a chat client and its logging functionality is meant to log those chat messages. This functionality should really be part of a chat bot, which I'm sure many out there exist to track stats. 

You could argue that chatterino's plugin API could do with some scheduling support which would allow you to send a system message every x minutes, subsequently showing up in the existing logs. If you think that would solve your issue, make a new discussion 👍

**Reply by pajlada** [reply]
With the current knowledge I have of your request, I agree with @jupjohn - if you want to elaborate on your use-case/needs we might reconsider.
